RULEngine/tracker.py

```diff
@@ -127,25 +133,38 @@ def game_state(self) -> Dict:
         return game_fields
 
     @property
-    def balls(self) -> List[Dict]:
-        return Tracker.format_list(self._balls, 'x y')
+    def balls(self) -> List[Position]:
+        active_balls = [b for b in self._balls if b.is_active]
+        return Tracker.format_ball(active_balls)
 
     @property
-    def blue_team(self) -> List[Dict]:
-        return Tracker.format_list(self._blue_team, 'x y orientation')
+    def blue_team(self) -> List[Pose]:
+        active_players = [p for p in self._balls if p.is_active]
+        return Tracker.format_team(active_players)
 
     @property
-    def yellow_team(self) -> List[Dict]:
-        return Tracker.format_list(self._yellow_team, 'x y orientation')
+    def yellow_team(self) -> List[Pose]:
+        active_players = [p for p in self._balls if p.is_active]
```

Comment:
ball instead of players

Reply:
(*&*&%?%&?*(&?%& fait une heure que je suis sur ce bug la